### DISCUSSION TITLE
Welcome to english Discussions!

### DISCUSSION THREAD

**Discussion by gomzyakov** [gomzyakov/github-achievements#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gomzyakov** [comment]
Hello everyone!

I'm Alexander Gomzyakov, a design-minded software engineer focused on building high-quality websites and applications. 

If you have any questions, wishes or suggestions, feel free to write about it here, in the discussions.

**Comment by galchenok** [comment]
Hello!

**Reply by neerajc0des** [reply]
hey, do you know any os website for people who can contribute using javascript?

**Reply by gomzyakov** [reply]
@Neeraj-2525 Hi!

There is a site https://good-first-issues.github.io which contains tasks for beginners (labeled `good first issie`). So far it contains only PHP issies, but I can improve it in the foreseeable future and add section with JS.

**Comment by OxYGC** [comment]
redme.md Can it be translated into Mongolian?

**Reply by gomzyakov** [reply]
Yes, of course) Can you do a PR with translation?
